README.md

```diff
@@ -291,7 +291,7 @@ For more information about the configuration format check [configuring section](
 
 # Upgrades
 
-Starting with version `0.16.0`, this operator supports upgrades to any newer
+Starting with version `0.42.0`, this operator supports upgrades to any newer
```

Comment:
No need to change it now, but consider adding a sentence about what is needed to upgrade from an older version.
(e.g. In order to upgrade from older versions, perform incremental upgrades through supported upgrade paths)

BTW, this is the reason of having this info in the RN may help, as there you can see the history.